### ISSUE TITLE
Security Benefit: MAXIMUM is confusing in red.

### ISSUE THREAD

**Issue by lgarron** [mozilla/wikimo_opsec#34]
Strawman alternative: "risk of not implementing".

<img width="644" alt="screen shot 2016-02-29 at 16 51 27" src="https://cloud.githubusercontent.com/assets/248078/13414092/b1c52c32-df04-11e5-9f4b-1f341aac5dcd.png">

From https://twitter.com/kangsterizer/status/704454522781839360

**Comment by gdestuynder** [comment]
@marumari I believe we should just replace it by the recommended configuration states. The document can discuss how useful each individual items are, but its more useful to have a list of what you need to have "anyway". That solves the level/labeling issue.

Eg (pretty inaccurate values, but just to get the point across)

**Modern**

- Redirect from HTTP
- Resource loading
- HSTS
- CSP
- COOKIES
[...]

**Intermediate**

- Redirect from HTTP
- HSTS
- Cookies
[...]

**old**
- robots.txt
[...]

**Comment by lgarron** [comment]
> @marumari I believe we should just replace it by the recommended configuration states. The document can discuss how useful each individual items are, but its more useful to have a list of what you need to have "anyway". That solves the level/labeling issue.

Everyone and their cousin makes lists like that. It's nice to see the wiki make a stand and provide digestible data in a table (apart from the coloring issue this but is about).

**Comment by marumari** [comment]
I agree with @lgarron, I don't like sorting things that particular way.  Especially since the needs don't line up with what are in the TLS guidelines.  For example, GitHub should use Intermediate TLS and Modern Web Security.  Mozilla.org should use Backwards Compatible TLS and Modern Web Security. And so on.  :)

**Comment by gdestuynder** [comment]
I strongly disagree. Here's why:

There are thousands of  Mozilla websites, each have a diff config because they liked setting A better than setting B. How do you tell if they're using safe settings?
If your answer is: "just scan them for what is safe", then the real answer is "follow standard recommended configurations" because that is *exactly* what the recommended configuration has to contain. 

Anything else you add to that is up to you of course. The standard recommended configuration *only* cares about ensuring you have the minimum that you always need for a defined target.
It may not contain anything that is not clearly making the website safer.

Note: its ok to have a bunch of additional docs that explain other options, and rationales for choosing which of course (and in fact, is a good idea).

In more simple words a pick-from-list achieves this:
![standards](https://cloud.githubusercontent.com/assets/956866/17631840/8eb95ac2-607a-11e6-9e1e-3d346e1ec015.png)

**Comment by jeffbryner** [comment]
I agree that red as a positive color to indicate 'yes you should do this and it's a good thing' is confusing. 

So as to not create another standard or scheme; how about changing the column header to '**importance**' or '**priority**' like we did in our POC test plan recently? Then it's more of a call out and red makes more sense. 

While usually I'm a fan of packaged controls, I think for configurations like this it makes sense to have a buffet of configuration options where folks can pick the ones that get them the score that they want as they progress. (i.e. I can't do CSP now, I'll do hsts,etc. )

**Comment by gdestuynder** [comment]
I think the new proposal in https://github.com/mozilla/wikimo_content/issues/50 solves this better now.

You can pick and use a score (web guidelines). You can use configuration states (TLS Guidelines).
Either way, you **must** get a risk level assigned that is standardized.

For the guidance document what this means is that the table would no longer show the "Security benefit", instead it would should which sites should use which:

Example, should/must use these (totally fictitious/random):

All sites: Redirection from HTTP, Resource loading, contribute.json, etc.
Maximum and high risk impact sites: CSP, cookies, etc.

Thoughts?

**Comment by gdestuynder** [comment]
the above PR has been reviewed and is now live
